### DISCUSSION TITLE
FaceSwap does not look like my model

### DISCUSSION THREAD

**Discussion by shortskirtlover** [lllyasviel/Fooocus#2267] [Q&A]
I am having issues with getting my generation to look like my FaceSwap, I have tried every setting I can think of and she just doesn't come out the way I would like. If any of you could give me suggestions I would appreciate it. I am using the same seed, resolution, no negative prompt, same styles and models, different settings in FaceSwap. I have also tried different checkpoints and LoRA as well. I don't know if I am overcomplicating things or what. 

CONTROL IMAGE:
![image (97)](https://github.com/lllyasviel/Fooocus/assets/159172418/1e4cde63-bc6e-46e5-aa41-a1b7f406096a)

IMAGE GUIDANCE SCALE = 5 / SHARPNESS = 2
FACESWAP = STOP 1 / WEIGHT .75
IMAGE PROMPT = STOP .5 / WEIGHT .5
![image - 2024-02-15T130819 753](https://github.com/lllyasviel/Fooocus/assets/159172418/52188416-cc3d-4199-a0d6-8e790eeedd15)

IMAGE GUIDANCE SCALE = 5 / SHARPNESS = 2
FACESWAP = STOP 1 / WEIGHT .75
PYRACANNY = STOP .8 / WEIGHT .5
![image - 2024-02-15T131533 568](https://github.com/lllyasviel/Fooocus/assets/159172418/a9e80380-a20a-4291-8124-5ab04ad0340e)

TOOK OUT PROMPT
IMAGE GUIDANCE SCALE = 5 / SHARPNESS = 2
FACESWAP = STOP 1 / WEIGHT .75
IMAGE PROMPT = STOP .8 / WEIGHT 1.5
![image - 2024-02-15T132659 535](https://github.com/lllyasviel/Fooocus/assets/159172418/2dd6a389-c4a9-491a-8444-2d615be0eded)

RAISED SHARPNESS
IMAGE GUIDANCE SCALE = 5 / SHARPNESS = 10
FACESWAP = STOP 1 / WEIGHT .75
PYRACANNY = STOP .5 / WEIGHT 1s
![image - 2024-02-15T134015 489](https://github.com/lllyasviel/Fooocus/assets/159172418/ff2d382d-6dd9-4093-adf1-a6d903be172c)

**Comment by ILikeToasters** [comment]
Increase weight for FaceSwap to 1 and try. Not sure if you have tried that but didn't see it in the ones you showed. I find best luck is above .9 weight.

**Comment by lgwjames** [comment]
im using a master photo of model neck up , stop at 9 and weight 8-855 and all ok

**Comment by infinity0** [comment]
Do not try to mix content generation and faceswap in one Generate step. Do your generation first, then apply the output as an input to the method described in #2644, i.e. Upscale + FaceSwap Stop At = 1.0.
